packages/itwin/measure-tools/src/measurements/DistanceMeasurement.ts

```diff
@@ -477,15 +478,26 @@ export class DistanceMeasurement extends Measurement {
     const dx = Math.abs(this._endPoint.x - this._startPoint.x);
     const dy = Math.abs(this._endPoint.y - this._startPoint.y);
 
+    let adjustedStart = this._startPoint;
```

Comment:
We should provide a helper function that is exported for this. I'm thinking we'll need to have the civil measurements also account for the global origin?